getMarkers.php

```diff
@@ -215,7 +216,7 @@
 		if(($property->getPart() == 'somata') && ($property->getRel() == 'in'))
 		{
 			$object_value = $property->getVal();
-			$soma_position = $soma_location["$object_value"];
+			array_push($soma_positions, $soma_location["$object_value"]);
```

Comment:
Any time we are getting these positions from $soma_location array in 184.. only thing we changed is from considering one value to n values (these might be unique like 1234 ) if we see on the UI CA1:Bucket++ one have all of them highlighted as they have 4 somatic positions in the db)